index.html

```diff
@@ -199,5 +199,77 @@ <h3 class="skill-title">Languages</h3>
         </ul>
       </div>
     </section>
+    <section id="contact-form">
+      <h3 id="contact-form-caption" class="heading text-secondary">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+      <form
+        action="https://formspree.io/f/mrgvbkea"
+        method="post"
+        id="contact-form"
+      >
+        <input
+          type="text"
+          max="30"
+          required
+          class="contact-input"
+          name="full-name"
+          placeholder="Full Name"
+        />
+        <input
+          id="email"
+          type="email"
+          class="contact-input"
+          required
+          name="email-address"
+          placeholder="Email address"
+        />
+        <textarea
+          placeholder="Enter text here"
+          name="message"
+          maxlength="500"
+          required
+          id="message"
+          cols="30"
+          rows="5"
+        ></textarea>
+        <button id="submit-button" class="button" type="submit">
+          Get in touch
+        </button>
+      </form>
+      <footer>
+        <ul class="social-icons">
+          <li class="social-icon">
+            <a href="https://facebook.com">
+              <i class="fa-brands fa-facebook"></i>
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="https://twitter.com">
+              <i class="fa-brands fa-twitter"></i>
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="https://linkedin.com">
+              <i class="fa-brands fa-linkedin"></i>
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="https://whatsapp.com">
+              <i class="fa-brands fa-whatsapp"></i>
+            </a>
+          </li>
+          <li class="social-icon">
+            <a href="mailto:bahirhakimy2020@gmail.com">
+              <i class="fa-solid fa-envelope"></i>
+            </a>
+          </li>
+        </ul>
+        <a class="footer-email" href="mailto:myemailhere@email.com"
+          >myemailhere@email.com</a
+        >
+      </footer>
```

Comment:
- [ ] Nice use of the `<ul>` tag to wrap the social media icons! 🚀 It would be even better if you add the links to your social media so people browsing through your portfolio (possible future recruiters) would have an easy way to connect with you! 🤓 🔧